app/app-text/views/reports/chm/aichi-target.json

```
@@ -0,0 +1,6 @@
+{
+    "fullTitle": "Full title",
+    "targetNumber": "Target Number ",
+    "strategicGoal":  "Strategic Goal",
+    "quickGuide":  "Quick Guide"
+}
```

the second word needs to be small
```suggestion
    "targetNumber": "Target number ",
    "strategicGoal":  "Strategic goal",
    "quickGuide":  "Quick guide"
}
```

---

fixed